app/model/SocialPlatform.scala

```diff
@@ -12,12 +12,22 @@ case object Facebook extends SocialPlatform {
   val name: String   = "Facebook"
 }
 
+case object Twitter extends SocialPlatform {
+  val domain: String = "twitter.com"
+  val name: String   = "Twitter"
+}
+
+case object Unknown extends SocialPlatform {
+  val domain: String = ""
+  val name: String   = "Unknown"
+}
+
 object SocialPlatform {
   implicit lazy val writes: Writes[SocialPlatform] = platform => JsString(platform.name)
 
-  def fromDomain(domain: String): SocialPlatform = {
-    domain match {
-      case Facebook.domain => Facebook
-    }
+  def fromDomain(domain: String): SocialPlatform = domain match {
+    case Facebook.domain => Facebook
+    case Twitter.domain  => Twitter
+    case _               => Unknown
```

Comment:
I know you probably wouldn't use it, but would it be worth capturing the unknown value? Maybe to reveal what they were later?

Reply:
It should come out in the UI so should be obvious.  I can't imagine how this case would happen though because each platform will be brought in one by one so should all be known.